README.md

```diff
@@ -1,10 +1,84 @@
-[![Test status](https://github.com/cdata/rs-ucan/actions/workflows/run_test_suite.yaml/badge.svg)](https://github.com/cdata/rs-ucan/actions/workflows/run_test_suite.yaml)
-[![Docs status](https://img.shields.io/docsrs/ucan)](https://docs.rs/ucan)
 [![Crate](https://img.shields.io/crates/v/ucan.svg)](https://crates.io/crates/ucan)
+[![codecov](https://codecov.io/gh/ucan-wg/rs-ucan/branch/main/graph/badge.svg?token=UZ53MKNKJC)](https://codecov.io/gh/ucan-wg/rs-ucan)
+[![Build status](https://github.com/ucan-wg/rs-ucan/actions/workflows/run_test_suite.yaml/badge.svg)](https://github.com/ucan-wg/rs-ucan/actions?query=)
+[![Docs](https://img.shields.io/docsrs/ucan)](https://docs.rs/ucan)
 [![License](https://img.shields.io/crates/l/ucan)](https://opensource.org/licenses/Apache-2.0)
+[![Discord](https://img.shields.io/static/v1?label=Discord&message=join%20us!&color=mediumslateblue)](https://discord.gg/JSyFG6XgVM)
+
+# rs-ucan
 
 This is a Rust library to help the next generation of web applications make use
 of UCANs in their authorization flows. To learn more about UCANs and how you
-might use them in your application, visit [https://ucan.xyz][ucan website]!
+might use them in your application, visit [https://ucan.xyz][ucan website] or
+read the [spec][spec].
```

Comment:
Thanks for the wonderful cleanup of this README. Would you be interested in knocking out https://github.com/ucan-wg/rs-ucan/issues/21 while you're at it? :crab:

Reply:
gotcha.